tasks/Shopping Plan.md

```diff
@@ -0,0 +1,32 @@
+# Summary
+>Individuals were assigned a number of tasks to complete alone and with a group to determine if there is a level of intelligence among groups as there is in an individual.
+
+# References
+> Source: https://science.sciencemag.org/content/330/6004/686.full
+> Supporting: https://science.sciencemag.org/content/sci/suppl/2010/09/29/science.1193147.DC1/Woolley_SOM.pdf
+
```

Comment:
Could we add E. McGrath, Groups: Interaction and Performance (Prentice-Hall, Englewood Cliffs, N], 1984) since tasks included in the supporting docs come from their work?